releaseNotes/JavaScriptProvider.md

```diff
@@ -2,6 +2,15 @@
 
 [Source code][source_code] | [Package (npm)][package] | [Samples][samples]
 
+## 1.0.0 - June 6, 2024
+
+**Release of the stable API version `v1`**.

```

Comment:
Usually we use the following format for stable version update:
`This is the first stable release of XYZ feature introduced in ABC version`

In this case, I think we can say that this is the first stable release of all the features that were released in previous preview versions.

Reply:
I think we should relist all features that made it to the stable release from the previous preview releases.

Reply:
updated. listed key features, and added a link to quickstart. See if we want to polish the wording or provide more details about the features.